concourse/model/traits/image_scan.py

```diff
@@ -76,6 +77,12 @@
         default=None,
         doc='protecode cfg name to use (see cc-config)',
     ),
+    AttributeSpec.optional(
+        name='cvss_version',
+        default=CVSSVersion.V2,
+        doc='CVSS version used to scan images',
```

Comment:
it is not used for scanning, but for evaluating the severity (i.d. whether the threshold is exceeded)